x/accounts/module.go

```diff
@@ -0,0 +1,165 @@
+package accounts
```

Comment:
this isnt implementing the appmodule yet or will it all?

Reply:
renamed to Keeper